docs/index.rst

```diff
@@ -1610,33 +1610,16 @@ It is possible to use factories in
 `PHPUnit data providers <https://phpunit.readthedocs.io/en/9.3/writing-tests-for-phpunit.html#data-providers>`_.
 Their usage depends on which Foundry version you are running:
 
-Data Providers with Foundry ^2.2
-................................
+Data Providers with Foundry ^2.2 and PHPUnit ^11.4
```

Comment:
Below, you have the extension requires 10+.

I think we should remove the version constraints from these subtitles and use a versionadded + warning for required phpunit versions

Reply:
I've updated the docs, WDYT?